device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/IotHubTransportTest.java

```diff
@@ -29,6 +29,7 @@
 /**
  * Unit tests for IotHubTransportPacket.
  */
+@SuppressWarnings("ThrowableNotThrown")
```

Comment:
is this because we annotate the test with the expected exception?

Reply:
I know that I explained this in another PR after you made this comment, so in case someone else sees this comment and is still confused:

> The warning is complaining because of how jmockit expects us to make a mocked method return by throwing an exception. In our tests, we do something like

>```
>someMockInstance.thisMethodShouldThrow();
>result = new IOException();
>```

> The warning sees that we create an exception instance, but never throw it and complains. This is the expected way for > jmockit to make mocked methods throw though, so I just suppressed these warnings